srcpkgs/leftwm/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'leftwm'
+pkgname=leftwm
+version=0.2.7
+revision=1
+archs="i686 x86_64"
```

Comment:
Do not limit archs.
If it is broken on musl add
```
if [ "$XBPS_TARGET_LIBC" = musl ] ; then
	broken="Some reason/upstream issue link"
fi
```

Reply:
alright. i am very new to packaging in void. I did not know that exists

Reply:
It isn't clear to me from your issue, did you use the package built from within xbps-src or did you build the software manually and try to use it? If the latter, you're probably not using the Void toolchain with the necessary patches.

Furthermore, please include a link to the issue in the broken definition.

Reply:
i did not try Void's toolchain. I once dual boot with x86_64-musl of Void and tried it for a bit (left it because there are some stuff I need to work properly), installed rustup and cargo from the website and build leftwm. it did not work. i assumed they were the same so I mentioned that it has trouble with musl, i was wrong. thanks for letting me know that Void has patches for it.

I guess I will remove that line and that if block then. and let the CI do its thing.